### ISSUE TITLE
Crash after restart

### ISSUE THREAD

**Issue by graasmund** [Xilmi/rotp-coder#1]
I get a crash when creating a new game directly after having played a while. Not easy to reproduce, so will amend this message if I find some more details. 

Now to the real reason for making this issue: This is - by far - the best and most faithful remake of MOO I have ever seen. I found it today, and have been more filled with happy nostagia than I thought possible. Very well done! Thank you!

**Comment by Xilmi** [comment]
Yes, that's a pretty old issue which I inherited from the base-version. I think it is related to not properly cleaning up AI-ship-designs or player-ship-designs. Not sure. It usually happens when you finished another game.
I never bothered to fix it because restarting the game was easy enough for me.

If you just want to praise the game, reddit.com/r/rotp is probably a better place for that. I mean I wouldn't even know where else you could have gotten this particular link otherwise. :D
